graphics/src/MeshManager.cc

```diff
@@ -138,6 +147,8 @@ const Mesh *MeshManager::Load(const std::string &_filename)
         extension.begin(), ::tolower);
     MeshLoader *loader = nullptr;
 
+    loader = &this->dataPtr->assimpLoader;
```

Comment:
we discussed at the Gazebo meeting that colladaLoader and objLoader might have some adhoc tags or internal modifications (in the daes or objs). We are thinking about adding a cmd flag to be able to use assimp with an argument but keep the current Collada and objloader as the default behaviour.

FYI @onurtore

Reply:
OK! That makes development on this PR considerably simpler since there won't be any risk to break existing behavior. Just to confirm the intention is also to keep STLLoader right? Therefore assimp will be used only for the new formats?

Reply:
I did't remember nothing about STLs

> Therefore assimp will be used only for the new formats?

Yes, that's the idea for now, or maybe add a CMD flag to allow to use ASSIMP. @onurtore how do you feel about openning a PR on top of this ?